### ISSUE TITLE
"Block pop-up windows" ignores exception list

### ISSUE THREAD

**Issue by merlin67** [bwinton/TabCenter#636]
I have several sites (including my banking site) which opens a new tab when I click links. TabCenter immediately closes these new tabs every time, even though I have the sites listed in my exception list.

**Comment by ericawright** [comment]
This is very odd, just to confirm, if you disable TabCenter the new tabs open correctly?

**Comment by merlin67** [comment]
Yes, they do, which is why it is so frustrating. I’ve had to shut off tabcenter because of it

**Comment by ericawright** [comment]
can you give me the steps to reproduce?

**Comment by merlin67** [comment]
well, that is difficult since the most common one is my banking site, and I can't exactly give you access to that... would have to think of another site it does it on.

**Comment by ericawright** [comment]
no, of course not. But perhaps I use the same site?
So I assume that this is the setting you have on that generally blocks the pop up windows?
from `about:preferences:
<img width="882" alt="screen shot 2016-09-27 at 10 25 02 am" src="https://cloud.githubusercontent.com/assets/10803178/18884528/a3b7f64a-849c-11e6-962a-11fb6a1d43af.png">
And you've added your bank to the exception list. Is there anything else you can think of that might be interacting with this?

**Comment by merlin67** [comment]
Well, the site is HSBCnet. Yes, that is the setting I am talking about, and my bank is in the exception list. I'm not a coder, so I don't know for sure, but one explanation for this may be the way the site opens pop ups. when I right-click on the link to see the information, it is a javascript command. would that have an effect?

**Comment by ericawright** [comment]
okay, last thing, can you let me know which link/button it is that you click on?

**Comment by merlin67** [comment]
Anything that goes to a different page, whether it is my "account details" page, or a bill payment page

**Comment by ericawright** [comment]
okay, thank you for the info, I'll see what I can do! (if you see it happen on any other site please let me know)

**Comment by merlin67** [comment]
Thanks, will do!
